views/base.njk

```diff
@@ -5,7 +5,8 @@
 {%- from 'checkboxes.njk' import checkBoxes as checkBoxes with context -%}
 {%- from 'buttons.njk' import formButtons as formButtons with context -%}
 {%- from 'input-file.njk' import fileInput as fileInput with context -%}
-{%- from 'benefit-result.njk' import benefitResult as benefitResult with context -%}
+{%- from 'benefit-result.njk' import blueLI as blueLI with context -%}
+{%- from 'benefit-result.njk' import infoLink as infoLink with context -%}
```

Comment:
is there a reason we call this in base, and not on the results page (since it seems to only be called  there)?

Reply:
Cuz I'm lazy

Reply:
Also I didn't realize importing on the results page would allow me to access them in the benefit result imports.

Reply:
Alright tested it it works and pushed up the new changes.